Packs/CommonScripts/ReleaseNotes/1_19_73.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### get-user-data
+
+- Added support for **additional_fields** argument that determines whether to add additional fields to the main fields or not.
```

Comment:
Added the **additional_fields** argument to control whether supplementary fields are included in the script's output. Set it to true to include them.